src/cogniteClient.ts

```diff
@@ -316,8 +317,8 @@ export default class CogniteClient {
    * const response = await client.get('/api/v1/projects/{project}/assets', { params: { limit: 50 }});
    * ```
    */
-  public get = (path: string, options?: BaseRequestOptions) =>
-    this.doRawRequest('get', path, options);
+  public get = <T = any>(path: string, options?: HttpRequestOptions) =>
```

Comment:
Breaking change: BaseRequestOptions is different from HttpRequestOptions. (enum vs string)

Reply:
Done